src/views/AddClientPage.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div>
+    <div class="columns">
+      <div class="column">
+        <div class="card">
+          <section class="fields">
+            <h1>New Client</h1>
+            <div class="field">
+              <p>This page is for creating new clients.</p>
+              <form @submit.prevent="go">
+              <div class="field">
+                <label>
+                  <span>Client Name</span>
+                  <input
+                    v-model="name"
+                    autocomplete=""
+                    placeholder="Name"
+                    required
+                  />
+                </label>
+              </div>
+              <div class="field">
+                <label>
+                  <span>Email</span>
+                  <input
+                    v-model="email"
+                    autocomplete=""
+                    placeholder="Email"
```

Comment:
Placeholder could make user think the field is already completed. We already have a `label` attached to each `input`, so placeholder can be removed.

https://medium.theuxblog.com/4-reasons-to-avoid-using-placeholder-text-in-forms-8fc33e724e4f